test/tests/meshgenerators/patch_mesh_generator/tests

```diff
@@ -0,0 +1,40 @@
+[Tests]
+  design = 'meshgenerators/PatchMeshGenerator.md'
+  issues = '#13453'
+  [./quad4]
+    type = 'Exodiff'
+    input = 'patch_mesh_generator.i'
+    cli_args = 'Outputs/file_base=quad4_out'
+    exodiff = 'quad4_out.e'
+    mesh_mode = 'replicated'
+    requirement = 'The system shall be able to create a patch test mesh with quad4 elements.'
+    recover = false
+  [../]
+  [./quad8]
+    type = 'Exodiff'
+    input = 'patch_mesh_generator.i'
+    cli_args = 'MeshGenerators/patch/elem_type=quad8 Variables/u/order=SECOND Outputs/file_base=quad8_out'
+    exodiff = 'quad8_out.e'
+    mesh_mode = 'replicated'
+    requirement = 'The system shall be able to create a patch test mesh with quad8 elements.'
+    recover = false
+  [../]
+  [./hex8]
+    type = 'Exodiff'
+    input = 'patch_mesh_generator.i'
+    cli_args = 'MeshGenerators/patch/elem_type=hex8 MeshGenerators/patch/dim=3 Outputs/file_base=hex8_out'
+    exodiff = 'hex8_out.e'
+    mesh_mode = 'replicated'
+    requirement = 'The system shall be able to create a patch test mesh with hex8 elements.'
+    recover = false
+  [../]
+  [./hex20]
+    type = 'Exodiff'
+    input = 'patch_mesh_generator.i'
+    cli_args = 'MeshGenerators/patch/elem_type=hex20 MeshGenerators/patch/dim=3 Variables/u/order=SECOND Outputs/file_base=hex20_out'
+    exodiff = 'hex20_out.e'
+    mesh_mode = 'replicated'
+    requirement = 'The system shall be able to create a patch test mesh with hex20 elements.'
```

Comment:
I'd prefer to see these "almost exactly the same"-type requirements done with the "requirement"+"detail" organization. It reduces the number of requirements we have and gives us bulleted lists in our requirements document (scroll to the bottom):
https://mooseframework.org/python/MooseDocs/generate.html